backend/src/notes/notes.service.ts

```diff
@@ -92,37 +92,52 @@ export class NotesService {
     owner: User | null,
     alias?: string,
   ): Promise<Note> {
+    // Check if new note doesn't violate application constraints
     if (alias) {
       this.checkNoteIdOrAlias(alias);
     }
-    const newNote = Note.create(owner, alias);
     if (noteContent.length > this.noteConfig.maxDocumentLength) {
       throw new MaximumDocumentLengthExceededError();
     }
-    //TODO: Calculate patch
+
+    // Create new note and set the initial properties
+    // We cast to a note early to keep the later code clean
+    const newNote = Note.create(owner, alias) as Note;
     newNote.revisions = Promise.resolve([
-      Revision.create(noteContent, noteContent, newNote as Note) as Revision,
+      //TODO: Calculate patch
+      Revision.create(noteContent, noteContent, newNote) as Revision,
     ]);
+
+    let everyoneAccessLevel;
+
     if (owner) {
+      // When we know an owner, an initial history entry is created
```

Comment:
Why are you describing this?

Reply:
It's true, however as it is not a simple one-line statement but instead three lines, I wouldn't say this needs to be removed explicitly.